cmd/migrate/main.go

```diff
@@ -48,6 +48,7 @@ func main() {
 	batch := flag.Int("batchLen", 500, "Specify how many chunks to read/write in one batch")
 	shardBy := flag.Duration("shardBy", 6*time.Hour, "Break down the total interval into shards of this size, making this too small can lead to syncing a lot of duplicate chunks")
 	parallel := flag.Int("parallel", 8, "How many parallel threads to process each shard")
+	metricsNamespace := flag.String("metrics.namespace", "cortex", "Namespace of the generated metrics")
```

Comment:
What's the reasoning behind this being configurable?
So that customers can change it back to `cortex` if the Loki 3.0 migration messes up their dashboards/alerts?

Reply:
Yes, that is it. This will be mentioned in the migration steps once the default is set to `loki` (in a future PR).

Reply:
Nice 👌